package.json

```diff
@@ -19,15 +19,20 @@
         {
             "name": "Travis Kosarek",
             "url": "https://github.com/traviskosarek"
+        },
+        {
+            "name": "Caleb Cartwright",
+            "url": "https://github.com/calebcartwright"
         }
     ],
-    "license": "ISC",
+    "license": "MIT",
     "private": true,
     "dependencies": {
         "valid-url": "^1.0.9",
         "vsts-task-lib": "^2.3.0"
     },
     "devDependencies": {
+        "@swellaby/nyc-config-complete-coverage": "0.0.4",
```

Comment:
This is our shared/reusable nyc config which I updated your nyc config to extend. It is literally the same as your existing, explicit values in the nyc property in this file, except that it also adds the `lcov` reporter which the external systems (sonar, coveralls, etc.) need to interpret code coverage

Reply:
Cool, I didn't know you had created one. I will use it from now on.